### ISSUE TITLE
Is the teacher model has trained parameter when load pretrained model? 

### ISSUE THREAD

**Issue by guoyilin** [megvii-research/ECCV2022-RIFE#284]
Hi, thanks for your exciting work,  my problem is when I finetune the pretrained model on my small data(mostly contain slow human action), the l1-loss is oscillation ,not decrease, I was wondering if the pretrained model (Loaded ArXiv-RIFE model) contained the trained teacher parameter? or maybe my data is too hard for train(contain the hand move)?

**Comment by hzwer** [comment]
<img width="388" alt="image" src="https://user-images.githubusercontent.com/10103856/195246054-21c89ffe-4b32-490e-a838-0e6e93e47607.png">
Hello, this is the result of my training on vimeo90K for 1000 epochs. It can be seen that the descending curve of the VFI model is very flat.

**Comment by guoyilin** [comment]
thanks, @hzwer ,  thanks for your reply, maybe i should contain training.
